src/Appwrite/Platform/Workers/Builds.php

```diff
@@ -242,7 +242,7 @@ protected function buildDeployment(callable $getFunctionsDevice, Func $queueForF
                     Console::execute('mkdir -p ' . $tmpDirectory . '/' . $rootDirectory, '', $stdout, $stderr);
 
                     // Merge template into user repo
-                    Console::execute('cp -rfn ' . $tmpTemplateDirectory . '/' . $templateRootDirectory . '/* ' . $tmpDirectory . '/' . $rootDirectory, '', $stdout, $stderr);
+                    Console::execute('cp -rfn ' . $tmpTemplateDirectory . '/' . $templateRootDirectory . '/. ' . $tmpDirectory . '/' . $rootDirectory, '', $stdout, $stderr);
```

Comment:
This should be sufficient.
We don't need to worry about copy .git or unwanted hidden files because we control the templates.